USER_MANUAL.html

```diff
@@ -349,8 +349,8 @@ <h2 id="tools---filter"><span class="header-section-number">9.1</span> Tools - F
 </thead>
 <tbody>
 <tr class="odd">
-<td>Noise Supression</td>
-<td>Enable noise supression, dereverberation, AGC of mic signal using the Speex pre-processor</td>
+<td>Noise Suppression</td>
+<td>Enable noise suppression, dereverberation, AGC of mic signal using the Speex pre-processor</td>
 </tr>
 <tr class="even">
 <td>700C/700D Auto EQ</td>
```

Comment:
The HTML version of the user manual is automatically generated from the Markdown version so best not to modify it directly.

Reply:
I just pushed an update to USER_MANUAL.md so that should correct itself hopefully.